Doppler.Jobs.Server/Startup.cs

```diff
@@ -74,15 +74,9 @@ public void ConfigureServices(IServiceCollection services)
                 sp.GetService<ILogger<DopplerCurrencyService>>(),
                 jobsConfig));
 
-            var dopplerSapServiceSettings = new DopplerSapServiceSettings();
-            Configuration.GetSection("DopplerSapService").Bind(dopplerSapServiceSettings);
-            services.AddSingleton(dopplerSapServiceSettings);
+            services.Configure<DopplerSapServiceSettings>(Configuration.GetSection("DopplerSapService"));
```

Comment:
```suggestion
            services.Configure<DopplerSapServiceSettings>(Configuration.GetSection(nameof(DopplerSapServiceSettings)));
```

Reply:
Done in this commit https://github.com/MakingSense/doppler-jobs/pull/2/commits/ad517b6d93d2cbe37742b20316b203812b511c8b